src/hooks/useSignalRMessages.ts

```diff
@@ -128,13 +128,15 @@ export function useSignalRMessages<
 
         /* Get all active messages */
         await connection.invoke('PeekMessages');
-      } catch (e: any) {
-        throw new Error('Connection failed: ', e);
+      } catch (error) {
+        console.error('Connection failed', error);
```

Comment:
Any reason this was changed from `throw new Error` to `console.error` ?

Reply:
https://github.com/equinor/amplify-components/blob/93e5a849a672bd323136bc97be8e7e82aaf2610e/src/api/core/CancelablePromise.ts#L115-L117

I looked at other cases where a `try catch` was implemented, and in those cases a `console` was used instead of `throw new Error`. I thought I would do the same while I was fixing the Eslint error anyways.